packages/@dcl/react-ecs/src/components/Dropdown/index.tsx

```diff
@@ -1,17 +1,21 @@
-import { PBUiDropdown } from '@dcl/ecs'
+import { PBUiDropdown, pointerEventsSystem } from '@dcl/ecs'
 
 import { parseProps } from '../utils'
 import { ReactEcs } from '../../react-ecs'
 import { EntityPropTypes } from '../types'
 import { UiDropdownProps } from './types'
+import { getFont, getTextAlign } from '../Label/utils'
 
 function parseUiDropdown(props: UiDropdownProps): PBUiDropdown {
+  const { textAlign, font, ...otherProps } = props
   return {
     acceptEmpty: false,
     options: [],
     selectedIndex: props.acceptEmpty ? -1 : 0,
     disabled: false,
-    ...props
+    ...otherProps,
+    ...getTextAlign(textAlign),
+    ...getFont(font)
```

Comment:
seems like we are repeating this in a bunch of files, is it possible to have a fn that just receives props and executes this stuff if the value exists?